backend/src/controllers/loginController.js

```diff
@@ -0,0 +1,82 @@
+import users from "../database/usersDb.js";
+import idGenerator from "../../unique-id.js";
+import fs from "fs";
+import { hashSync, compareSync } from "bcrypt";
+import jwt from "jsonwebtoken";
+import dotenv from "dotenv";
+dotenv.config();
+import { fileURLToPath } from "url";
+import path from "path";
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+const filePath = path.join(__dirname, "../../../frontend/views/login.html");
+
+const accessKey = process.env.ACCESS_TOKEN_SECRET;
+const uniqueId = idGenerator();
+
+const getLogin = (req, res) => {
+  res.sendFile(filePath);
+};
+
+const postLogin = (req, res) => {
+  const { username, password } = req.body;
+  const user = users.find((user) => user.username === username);
+
+  if (!user) {
+    return res.status(400).json({ message: "user does not exist" });
```

Comment:
For authentication errors, you should return 401. Try to read about the various error codes.